mariadb/docker-entrypoint-initdb.d/.gitignore

```diff
@@ -1 +0,0 @@
-*.sql
```

Comment:
@denis-chmel 
1. it was removed here https://github.com/shoxy/laradock/commit/aa6821c14e347c144e586de76e8a98031681c8b8 by @therealartz 
2. I guess it was removed because we need to keep this script https://github.com/shoxy/laradock/commit/aa6821c14e347c144e586de76e8a98031681c8b8#diff-28c6bad175b5cde197ccd3ff548edd52115b5a92bcf0488a3e84cbbe5ebb5308 in the repo, because it creates all needed DBs.

As an option we can keep an empty gitignore file, do we need that?